rust/src/lib.rs

```diff
@@ -280,19 +368,14 @@ impl StableDiffusion {
             }
         }
     }
-    pub fn set_lora_model_dir(&mut self, lora_model_dir: &str) -> &mut Self {
-        {
-            self.lora_model_dir = lora_model_dir.to_string();
-        }
-        self
-    }
 }
 impl<'a> BaseFunction<'a> for TextToImage<'a> {
     fn base(&mut self) -> &mut BaseContext<'a> {
         &mut self.common
     }
     fn generate(&self) -> Result<(), WasmedgeSdErrno> {
         if self.common.prompt.is_empty() {
+            println!("self.common.prompt.is_empty()\n\n\n\n");
```

Comment:
what's the purpose of this statement? If it is for debugging, please remove it. Thanks!